.buildkite/pipeline.yml

```diff
@@ -21,6 +21,6 @@ steps:
       npm install
       npm run test
       npm run build:ts
-      if [[ $(echo $BUILDKITE_MESSAGE | tr '[:upper:]' '[:lower:]') =~ release ]];then
```

Comment:
I think this line checks if the message includes the word "release" to know if to have the release option. By changing it to check if it's not a PR means you'll have the release button on any build (even though you won't be able to actually release since the script checks if you're on 'release' branch...)

Reply:
I don't think so.. 
What determines if to show the release button is at the top of the file. 
This is how the pipeline of uilib looks also

Reply:
Ok